src/bin/admin.rs

```diff
@@ -0,0 +1,110 @@
+use std::time::Duration;
+
+use distrib_kv_store::raft_node::RaftNode;
+use distrib_kv_store::start_example_raft_node;
+
+use distrib_kv_store::carp::Carp;
+use std::collections::HashMap;
+
+use std::fs;
+use std::error::Error;
+
+use toml;
+use serde::Deserialize;
+use serde_json;
+
+#[derive(Deserialize)]
+struct Config {
```

Comment:
Should we add IP addresses to Config struct instead of hard-coding?

Reply:
this is a good idea, will do